lib/samples/set_initial_viewpoint/set_initial_viewpoint.dart

```diff
@@ -0,0 +1,85 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import 'package:arcgis_maps/arcgis_maps.dart';
+import 'package:flutter/material.dart';
+
+class SetInitialViewpoint extends StatefulWidget {
+  const SetInitialViewpoint({super.key});
+
+  @override
+  State<SetInitialViewpoint> createState() => _SetInitialViewpointState();
+}
+
+class _SetInitialViewpointState extends State<SetInitialViewpoint> {
```

Comment:
I think we still want `with SampleStateSupport` and the import

Reply:
Added the `SampleStateSupport` import.